src/poliastro/twobody/orbit/creation.py

```diff
@@ -1,223 +1,29 @@
-from typing import List, Union
 from warnings import warn
 
 import numpy as np
-from astropy import time, units as u
-from astropy.coordinates import (
-    ICRS,
-    CartesianDifferential,
-    CartesianRepresentation,
-    get_body_barycentric,
-)
+from astropy import units as u
 
 from poliastro.constants import J2000
 from poliastro.frames import Planes
 from poliastro.frames.util import get_frame
-from poliastro.threebody.soi import laplace_radius
 from poliastro.twobody.elements import (
-    coe2rv_many,
-    eccentricity_vector,
-    energy,
     get_eccentricity_critical_argp,
     get_eccentricity_critical_inc,
     get_inclination_critical_argp,
     heliosynchronous,
-    hyp_nu_limit,
-    t_p,
 )
 from poliastro.twobody.mean_elements import get_mean_elements
-from poliastro.twobody.propagation import farnocchia, propagate
-from poliastro.twobody.sampling import sample_closed
-from poliastro.twobody.states import (
-    BaseState,
-    ClassicalState,
-    ModifiedEquinoctialState,
-    RVState,
-)
-from poliastro.util import find_closest_value, norm, wrap_angle
-from poliastro.warnings import OrbitSamplingWarning, PatchedConicsWarning
-
-try:
-    from functools import cached_property  # type: ignore
-except ImportError:
-    from cached_property import cached_property  # type: ignore
-
-
-ORBIT_FORMAT = "{r_p:.0f} x {r_a:.0f} x {inc:.1f} ({frame}) orbit around {body} at epoch {epoch} ({scale})"
-# String representation for orbits around bodies without predefined
-# Reference frame
-ORBIT_NO_FRAME_FORMAT = (
-    "{r_p:.0f} x {r_a:.0f} x {inc:.1f} orbit around {body} at epoch {epoch} ({scale})"
-)
+from poliastro.twobody.states import ClassicalState, ModifiedEquinoctialState, RVState
+from poliastro.util import find_closest_value
 
 
-class Orbit:
-    """Position and velocity of a body with respect to an attractor
-    at a given time (epoch).
-
-    Regardless of how the Orbit is created, the implicit
-    reference system is an inertial one. For the specific case
-    of the Solar System, this can be assumed to be the
-    International Celestial Reference System or ICRS.
-
+class OrbitCreationMixin:
```

Comment:
I don't see ourselves reusing this for Orbit arrays (the logic will be different) so maybe we can rename it?

...On the other hand, the class is not called `ScalarOrbit`, but `Orbit`. So, let's keep it.